demo/app/src/components/adsSizes.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Block } from "baseui/block";
+import AdSize from "./adSize";
+
+const AdsSizes = props => {
+  const { breakpoint, adsSizes, setSelection } = props;
+
+  return (
+    <React.Fragment>
+      <label>{breakpoint}</label>
+      <Block
+        display="grid"
+        gridTemplateColumns="repeat(auto-fill, minmax(100px, 1fr))"
+        gridAutoRows="minmax(30px, auto)"
+        gridGap="10px"
+      >
+        {adsSizes.map((ad, i) => (
+          <AdSize
+            {...ad}
+            key={i}
```

Comment:
Using indexes as `key` is an anti pattern in react. It's not catastrophic but it can have performance implications. If these add sizes have a unique value on one of their keys that should be used as the value to prevent unnecessary re rendering.

Reply:
Correct. Stephen and I discussed this, in this case, it is fine to use index cos this is static data, but as a best practice, I changed this to a unique value: 
`key={breakpoint + ad.h + 'x' + ad.w}`